spot-contracts/contracts/FeePolicy.sol

```diff
@@ -245,12 +246,15 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
         flashRedeemFeePercs = flashRedeemFeePercs_;
     }
 
-    /// @notice Updates the rebalance reaction lag.
-    /// @param maxPerpDebasePerc_ The magnitude of the daily debasement.
-    /// @param maxPerpEnrichPerc_ The magnitude of the daily enrichment.
-    function updateRebalanceRate(uint256 maxPerpDebasePerc_, uint256 maxPerpEnrichPerc_) external onlyOwner {
-        maxPerpDebasePerc = maxPerpDebasePerc_;
-        maxPerpEnrichPerc = maxPerpEnrichPerc_;
+    /// @notice Updates the rebalance rate.
+    /// @param debasementSystemTVLPerc_ The percentage of system tvl out of perp on debasement.
+    /// @param enrichmentSystemTVLPerc_ The percentage of system tvl into perp on enrichment.
+    function updateMaxRebalancePerc(
```

Comment:
`updateRebalanceRates` is probably best, given the names moved away from the "max" terminology. We can keep these inline.

Reply:
done 👍